src/monitors/WhammMonitor.v3

```diff
@@ -347,6 +354,56 @@ class WhammMonitor(named_modules: Array<NamedModule>) extends Monitor {
 		}
 		return args;
 	}
+	private def getNumTargets(op: Opcode, cv: CodeValidator, expType: ValueType) -> WhammArg {
+		var arg: WhammArg;
+		match (op) {
```

Comment:
I think this will be a little less nested if you first check `op != Opcode.BR_TABLE` and return early.